www/js/controllers.js

```diff
@@ -55,7 +38,32 @@ angular.module('quizmaster.controllers', [ 'ngActionCable' ])
     $scope.openModal = function () {
       $scope.modal.show();
       consumer.subscribe(callback)
-        .then(function(data) {
+        .then(function() {
+          this.submitAnswer = function() {
+            var dataset, quiz, answer, question, team, answer_hash ;
+            dataset = angular.element(document.querySelector('#info'))[0];
+            quiz = dataset.getAttribute('data-quiz-id');
+            // team = dataset.getAttribute('data-team-id');
+            team = 1;
+            question = dataset.getAttribute('data-question-id');
+            answer = angular.element(document.querySelector('#body'))[0].value;
+
+            var sendAnswer = function() {
+              consumer.send(answer_hash, 'submit_answer');
+            };
+
+            if (answer.trim().length >= 1) {
+              // Team ID needs to come from cookies or something here. - hard-coded as 1 for the moment
+              answer_hash = {answer: answer, team_id: team, quiz_id: quiz, question_id: question};
```

Comment:
I think it may be even easier than that. We never leave the page - the Quiz page that receives cables is a modal - so we possibly just need to set a variable when the team logs in and then we should be able to access it continuously.